### PR TITLE
SC Stupid Sized Map Update

### PR DESCRIPTION
**Pull request by Woodratt**
Or how Woodrat spent too long on something that was supposed to be simple.

- Addition of a Explorer Radio Channel and headsets.
- Explorers and Search and Rescue job slots added
- Adjustments to the pilot job including getting rid of the flatcap
- Map fixes, adjustments, (including signs pointing to cryo) planetside side map additions splitting up of the wilderness into two area sections
- Disabled lighting on the arrivals shuttle area once it is docked with the station, should help with the whole issue of lighting bugging out and having dark space
- Fixed EMS jacket missing icons
- Fixed Research signs being missing for some reason, addition of directional signs for cryo
- Addition of a Search and Rescue Winter coat
- Probably a dozen other small bug fixes I forgot, and bug additions

Tested, seems fine. At least in short tests.

### CODE REVIEW

File: code/controllers/communications.dm [hunk 1]
```diff
@@ -137,6 +138,7 @@ var/list/radiochannels = list(
 	"Raider"		= RAID_FREQ,
```
Comment (mistyLuminescence) [inline code/controllers/communications.dm]: the fact that some of these are indented and some of them aren't causes me literal pain

File: code/game/jobs/jobs.dm [hunk 2]
```diff
@@ -41,10 +41,8 @@ var/const/CARGOTECH			=(1<<7)
 var/const/MINER				=(1<<8)
```
Comment (mistyLuminescence) [inline code/game/jobs/jobs.dm]: i live in constant agony from indentation

File: code/modules/mob/living/say.dm [hunk 3]
```diff
@@ -31,6 +32,7 @@ var/list/department_radio_keys = list(
 	  ":U" = "Supply",		".U" = "Supply",
 	  ":V" = "Service",		".V" = "Service",
```
Comment (mistyLuminescence) [inline code/modules/mob/living/say.dm]: glass bones and paper skin, it works but it hurts etc

File: maps/southern_cross/structures/closets/misc.dm [hunk 4]
```diff
@@ -1,15 +1,94 @@
...
+
+	New()
+		..()
+		if(prob(50))
+			new /obj/item/weapon/storage/backpack(src)
+		else
+			new /obj/item/weapon/storage/backpack/satchel/norm(src)
+		new /obj/item/clothing/under/explorer(src)
+		new /obj/item/clothing/suit/storage/hooded/explorer(src)
+		new	/obj/item/clothing/mask/gas/explorer(src)
```
Comment (mistyLuminescence) [inline maps/southern_cross/structures/closets/misc.dm]: every morning i break my legs and every afternoon i break my arms

File: maps/~map_system/maps.dm [hunk 5]
```diff
@@ -173,3 +173,22 @@ var/list/all_maps = list()
 	if (using_map.zlevels["[z]"] == src)
 		using_map.zlevels -= "[z]"
 	return ..()
+
+// Access check is of the type requires one. These have been carefully selected to avoid allowing the janitor to see channels he shouldn't
+// This list needs to be purged but people insist on adding more cruft to the radio.
+/datum/map/proc/default_internal_channels()
```
Comment (Leshana) [inline maps/~map_system/maps.dm]: Is it intentional that default_internal_channels() proc is added but never used anywhere?

**Pull request merged**